src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -93,6 +93,13 @@
  *  Inet4Address.ofLiteral("02130706689"); // ==> /127.0.1.1
  * }
  *
+ * <p> The above forms adhere "strict" decimal-only syntax.
+ * Additionally, the {@link Inet4Address#ofPosixLiteral(String)}
+ * method implements a POSIX {@code inet_addr} compatible "loose"
+ * parsing algorithm, allowing octal and hexadecimal address segments.
+ * Please refer to <a href="https://www.ietf.org/rfc/rfc6943.html#section-3.1.1">
+ * <i>RFC&nbsp;6943: Issues in Identifier Comparison for Security
+ * Purposes</i></a>.
  * <p> For methods that return a textual representation as output
  * value, the first form, i.e. a dotted-quad string, is used.
```

Comment:
Maybe something like this?

```suggestion
 * <p> For methods that return a textual representation as output
 * value, the first form, i.e. a dotted-quad string in strict decimal notation, is used.
```

Reply:
Yes, I think this makes it clear.

Reply:
accepted.